build.gradle

```diff
@@ -65,7 +65,6 @@ apply from: 'gradle/ide.gradle'
 apply from: 'gradle/forbidden-dependencies.gradle'
 apply from: 'gradle/formatting.gradle'
 apply from: 'gradle/local-distribution.gradle'
-apply from: 'gradle/fips.gradle'
```

Comment:
Can you explain why we no longer want the `fips.gradle` file? I would expect us to now always run it perhaps, but instead you remove it.

Reply:
Note; I'm ok with removing this - but would love to have more context on the impact of its removal.

Reply:
I believe the primary purpose of `fips.gradle` is to replace the standard BC libraries on the classpath with the BC-FIPS libraries. Since the standard BC libraries can be entirely excluded, I suggest controlling this process with an additional system parameter like `OPENSEARCH_FIPS_ENABLE_JVM=true`, or by performing the same task with a gradle init script. This "process" involves exchanging the security policy files, setting the necessary JVM parameters, and managing the initial internal keystore password. This means the entire test suite would run with FIPS enabled or disabled based on a single parameter (or additional script), without the option to switch between these modes during runtime.

Reply:
Thanks - One other way that I am thinking about this - with this additional option, does it mean that we should start creating an alternative distribution of OpenSearch that is FIPS compliant?  E.g. are you imagining consuming this change by building the repo on your own?